bin/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Tag, Push and Deploy only if it's not a pull request
+if [ -z "$TRAVIS_PULL_REQUEST" ] || [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
+
+    # Push only if we're testing the staging branch
```

Comment:
We ought to leave this enabled by default

Reply:
Agreed. See notes below, but recommendation is that you replace it with something from your working example.

Reply:
check re-enabled in last commit